### PR TITLE
Revisions to notebook and console menus

### PR DESCRIPTION
**Pull request by ellisonbg**

### CODE REVIEW

**Comment by ellisonbg** [comment]
Fixes #1443

**Comment by blink1073** [comment]
This has a conflict from #1636; the console item should be renamed to reflect Halt->Shutdown as well.

**Comment by ellisonbg** [comment]
OK I have rebased and updated everything. Should be good to go!

File: src/console/plugin.ts [hunk 1]
```diff
@@ -263,7 +257,32 @@ function activateConsole(app: JupyterLab, services: IServiceManager, rendermime:
...
+  command = CommandIDs.closeAndShutdown;
+  commands.addCommand(command, {
+    label: 'Close and Shutdown',
+    execute: () => {
+      let current = tracker.currentWidget;
+      if (current) {
+        console.log('Close and Shutdown for Consoles not implemented.')
+      }
+    }
+  });
```
Comment (blink1073) [inline src/console/plugin.ts]: Missing a palette entry for this one.

Comment (blink1073) [inline src/console/plugin.ts]: Oh, nevermind, I see the commit, will open a new issue.

**Comment by blink1073** [comment]
Thanks!

**Pull request merged**